Client/Frontend/AuthenticationManager/PasscodeEntryViewController.swift

```diff
@@ -16,20 +16,32 @@ import SwiftKeychainWrapper
 /// Presented to the to user when asking for their passcode to validate entry into a part of the app.
 class PasscodeEntryViewController: BasePasscodeViewController {
     weak var delegate: PasscodeEntryDelegate?
-    fileprivate let passcodePane = PasscodePane()
-
+    fileprivate var passcodePane: PasscodePane?
+    
+    override init() {
+        super.init()
+        passcodePane = PasscodePane(title:nil, passcodeSize:authenticationInfo?.passcode?.count ?? 6)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
Ug. I hate this pattern. No change needed, I just hate every line of these constructors. Especially `aDecoder`.

Reply:
agreed.